frontend/src/FilterLists/ImportFilterList.tsx

```diff
@@ -0,0 +1,46 @@
+
+import { Intent, MenuItem } from '@blueprintjs/core';
+import { useState } from 'react';
+
+import { ImportFilterList as ImportFilterListBackend } from '../../wailsjs/go/files/Files';
+import { AppToaster } from '../common/toaster';
+
+
+export function ImportFilterList({ onAdd }: { onAdd: () => void }) {
+  const [loading, setLoading] = useState(false);
+
+  const handleImport = async() => {
+    setLoading(true);
+
+    try {
+      const result = await ImportFilterListBackend();
+
+      const blob = new Blob([JSON.stringify(result)], { type: 'application/json' });
+      const url = URL.createObjectURL(blob);
+      const link = document.createElement('a');
+      link.href =  url;
+      link.download = 'filter-lists.json';
+      document.body.appendChild(link);
+      link.click();
+
+      document.body.removeChild(link);
+      URL.revokeObjectURL(url);
```

Comment:
`result` is always null since the backend function only returns an error. Everything apart from the function call looks redundant.